README.md

```diff
@@ -86,8 +84,9 @@ The figure below shows an example of a simple provenance graph, that combines th
 
 ![Provenance](docs/figs/provenance.png)
 
-Given the network of `prov:used` and `prov:wasInformedBy` relations, it is possible to
-infer `prov:wasInformedBy` and `prov:wasDerivedFrom` relations.
+To make it easier to see where concepts belong, have concepts belonging to the PINK namespace in the figure above been written in cursive.
+
+Given the network of `prov:used` and `prov:wasInformedBy` relations, it is possible to infer `prov:wasInformedBy` and `prov:wasDerivedFrom` relations.
```

Comment:
```suggestion
Given the network of `:used` and `:wasInformedBy` relations, it is possible to infer `:wasInformedBy` and `:wasDerivedFrom` relations. This behaviour is inherited from the parent classes in prov. 
```

Reply:
No, PROV-O does not offer such reasoning. That is something we have added using SWRL rules.